### PR TITLE
renames social post module

### LINKED ISSUE #62: Rename social post module
All the submodules in this repo are namespaced with `localgov_elections_*` execpt the social post module with uses `localgov_election_...` (singular).

### PR DESCRIPTION
**Pull request by markconroy**
Closes #62 

===
Thanks to [Big Blue Door](https://www.bigbluedoor.net/) for sponsoring my time to work on this.

### CODE REVIEW

**Review by dedavidson — APPROVED** [review]
Tested locally and all works.

Assume existing installs will need to uninstall sub-module before applying update? So it will be "breaking" change?

**Comment by markconroy** [comment]
@dedavidson I think we probably know who is using this module at the moment, so maybe message each to ask them to uninstall it if they have it installed?

Failing that, I guess we can come up with a step-by-step

1. mark old module deprecated
2. uninstall old module
3. install new module
4. next release remove old module

**Comment by dedavidson** [comment]
@markconroy sounds like a plan. Just need to make clear in release notes and promote on slack

**Comment by markconroy** [comment]
PR updated to 

1. Leave the original module in place
2. Mark the original module as deprecated
3. Uninstall the original module if the new module is being installed

===
Thanks to [Big Blue Door](https://www.bigbluedoor.net/) for sponsoring my time to work on this.

**Comment by finnlewis** [comment]
@dedavidson I was just commenting on the issue, see https://github.com/localgovdrupal/localgov_elections_reporting/issues/62#issuecomment-2174255920

Might we want to take this opportunity to rename modules to be consistent with parent / child naming conventions? 

localgov_elections_reporting
localgov_elections_reporting_soical_posts

or even rename the parent module to 

localgov_elections
localgov_elections_social_posts

**Comment by willguv** [comment]
I'm in favour of localgov_elections if it's not too much work

**Comment by dedavidson** [comment]
I'd prefer to keep reporting or something similar in the name as the module doesn't run elections it just reports on them.

**Comment by finnlewis** [comment]
Discussing in Merge Tuesday, we would like to have a breakout meeting about this. 

Do we have time to rename things now and save tech debt?

File: modules/localgov_elections_reporting_social_post/localgov_elections_reporting_social_post.info.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: LocalGov Election Reporting Social Post Integration
+description: Social API integration
+type: module
+package: Localgov Elections Reporting
```
Comment (dedavidson) [inline modules/localgov_elections_reporting_social_post/localgov_elections_reporting_social_post.info.yml]: This needs to be changed to "LocalGov Drupal"

**Review by dedavidson — CHANGES_REQUESTED** [review]
Package for new module needs changed but otherwise all good from local testing

**Comment by markconroy** [comment]
Hi @dedavidson 

That change is made now, can you give this one final check and mark it approved if you're happy with it.

**Review by dedavidson — APPROVED** [review]
All good.

**Pull request merged**